roles/network_plugin/calico/templates/calicoctl.kdd.sh.j2

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+DATASTORE_TYPE=kubernetes \
+KUBECONFIG={{ kube_config_dir }}/admin.conf \
```

Comment:
It just came to my attention that calicoctl.sh wouldn't work on non-master nodes as `{{ kube_config_dir }}/admin.conf` is not deployed on them.

Reply:
I have cherry-picked the changes done by @jlacoline, which delegates the calicoctl.sh calls to the master.